packages/tasit-contracts/.secret.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@marcelomorgado Did you mean for this to be `.gitignore`d?

Reply:
I've added this file to the `.gitignore`, I think that is being shown because I added after the commit of the `.secret.json`

Reply:
Ah ok - you may have to manually remove it then

Reply:
Done.

Reply:
Thanks! Resolving